### PR TITLE
Create civilianpeterparker

### PR DESCRIPTION
**Pull request by SirRicardo**

### CODE REVIEW

**Comment by MrCraft888** [comment]
I think you need the regular skin too

**Comment by malonnnn** [comment]
the file should be named "civilianpeterparker.ms" not "civilianpeterparker"

File: CommandHelper/LocalPackages/commands/buffs/civilianpeterparker.ms [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@plr = player()
+
+
+
+@hero = associative_array(
+            'name': '§4Civilian§fPe§4ter§0P§4ar§0k§4er',
+            'class': 'civilianpeterparker',
+            'sig': '
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/civilianpeterparker.ms]: missing the sig and val here

File: CommandHelper/LocalPackages/commands/buffs/civilianpeterparker.ms [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            'class': 'civilianpeterparker',
+                     'sig': 'KpeakiLTqI1ZmndDyEK1lxMv4ev1o7qOUZUFEvIp0FtcDuXEkjsiPptOqE6d9A9WUd+2nZ+72ks/qEs4IBouv7O3SXUZwaNH/E12WkmeTiuEFwyLAWmXRoTZEoCxpCbwJ/r41XKHM5LuLY3KqgepsDkPJ+gTlwDdHAPvnP+JUrzBYqolo01D65sNmLxjGXKuzTPcO5MPjyVG8k92PogubIuPofcPojakMTUnH8U05Y8Dwo9mf70mqGrzLz/aIA6389a3VfAD+Yfv5B41Sp1Fx2sE29SE8zx5d9ZnKFkLU14NTNtetbYrbc0J8QnNtxNmVuW38B1a9SNzCpqt/EV3xKrsUUWSJAfsR5TX2FUTPz3VeYvhlj8mGJb6eOi/aOpRJYTQrV0T5f+XY9ULePvJMWcQuqE0NNedIxTIfbc4ZG/3onDKlN0/rueS5tCgHuVDPWnwTJ2P5Y9K2BnUMSmdA7NsQsjA0eQhTr7tb9Ci6sEVLiIkLMKorLJ8H5bp1BbCnbEsr0V8I8W1tNQIAUHT3GU+uOdkn+iwn3fx6fa4Lhtry4XFgJxgSZfamQ3OaAnIOKhtutWp6cwKaWHfPEa0ohYWsXCAmY9Q6GLKua7D37vfP44e5qo41zb1bbfETEecrsgATcdp4hoEBhSMDtquP00C4oOxjPTXFkDE8OFmhmk=',
+                     'val': 'eyJ0aW1lc3RhbXAiOjE0OTE1MDM2NTExODcsInByb2ZpbGVJZCI6ImJmYzllNDRkNGU3MDQ1NDQ4NjI4ZGY5Y2U5YmUxYmZlIiwicHJvZmlsZU5hbWUiOiJTaXJSaWNhcmRvIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS82OTdiZDJhNWY3ZjYxMDMyNDRlMDRlNmM3YzczOWNkNmZmMzJiMmFiMTU0Yjc1YzAyZTliOTEyYmY3NzY1MjcifX19',
+            ),
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/civilianpeterparker.ms]: why is there a ) here

File: CommandHelper/LocalPackages/commands/buffs/civilianpeterparker.ms [hunk 3]
```diff
@@ -0,0 +1,28 @@
+@plr = player()
+
+
+
+@hero = associative_array(
+            'name': '§4Civilian§fPe§4ter§0P§4ar§0k§4er',
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/civilianpeterparker.ms]: The name should just be Peter Parker

**Pull request closed without merging**